regulations/tasks.py

```diff
@@ -0,0 +1,59 @@
+from __future__ import absolute_import
+
+import os
+import shutil
+import tempfile
+import contextlib
+
+import boto3
+import requests
+from celery import shared_task
+from django.conf import settings
+
+
+@shared_task
+def submit_comment(sections):
+    with TemporaryDirectory() as path:
+        files = [
+            fetch_file(path, file['key'], file['name'])
+            for section in sections
+            for file in section.get('files', [])
+        ]
+        comment = build_comment(sections)
+        requests.post(
+            settings.REGS_API_URL,
+            data={
+                'comment': comment,
+            },
+            files=[
+                ('file', open(file, 'rb'))
+                for file in files
+            ],
```

Comment:
It's quite possible that `requests` is just materializing this behind the scenes :shrug: